Test/Platform/Tests/CLR/System/Text/Regexp/GroupTests.cs

```diff
@@ -1,4 +1,4 @@
-using System;

```

Comment:
This file is not related to the enc28j60 driver.
But GroupTests.cs file has ANSI encoding and TM special character so in non-western, it occurs compile problem. I changed it to Unicode. Should I have to separate it?